supervisor/addons/build.py

```diff
@@ -103,25 +96,40 @@ def additional_labels(self) -> dict[str, str]:
         """Return additional Docker labels."""
         return self._data[ATTR_LABELS]
 
-    @property
-    def is_valid(self) -> bool:
+    def dockerfile(self) -> Path:
```

Comment:
I'd prefer if we name such property replacement with a verb, e.g.

```suggestion
    def get_dockerfile(self) -> Path:
```

This also has the advantage if we were to use them as a property still, not end up with unnoticed surprises.

Reply:
Yea that's fair. I'll have to do that to a bunch of these, ill scan it 👍

Reply:
Oh actually this is the only one. I have a follow-up branch already finishing this effort, I think I'm mixing up the two. I'll make sure to correct any instances of this in the follow-up branch.